Longest word in Trie Tree Gotsev lab 1

Pavel36 opened the pull request.

Pavel36 committed:
Add files via upload

Pavel36 committed:
Merge pull request #1 from Pavel36/PavelGotsev-patch-1

Pavel Gotsev Longest word in trie tree

tapoton reviewed (changes requested):
Добавил в дерево слово austroneerqwer, самым длинным осталось слово austroneer

Pavel36 committed:
Update Source.cpp

Изменил проверку в функции longest, теперь при вводе слова austroneerqr, возвращает это же слово, как самое длинное

Pavel36 commented:
исправил ошибку

tapoton commented:
Хэш-таблицы: Задан набор записей следующей структуры: табельный номер, ФИО, заработная плата. По табельному номеру найти остальные сведения.
